docs/signature.md

```diff
@@ -0,0 +1,93 @@
+# Image Signature Specification
+
+**Version 0.1**
+
+## Introduction
+
+This document defines a detached container image signature object and signing methods.
+
+## Signature Format
+
+```js
+{
+    "critical": {/* required fields */
```

Comment:
> All signature blobs MUST start with the ASCII bytes representing their media type and a trailing newline.

Sorry, this should have been:

> All signed-assertion blobs MUST start with the ASCII bytes representing their media type and a trailing newline.

Presumably the *signature* blobs (e.g. a detached OpenPGP signature) have already taken steps to avoid the “tricked you into interpreting this as the wrong protocol/schema” vulnerability.